src/tsdb/inc/tsdbFile.h

```diff
@@ -38,7 +38,7 @@
 #define TSDB_FILE_IS_OK(tf) (TSDB_FILE_STATE(tf) == TSDB_FILE_STATE_OK)
 #define TSDB_FILE_IS_BAD(tf) (TSDB_FILE_STATE(tf) == TSDB_FILE_STATE_BAD)
 
-typedef enum { TSDB_FILE_HEAD = 0, TSDB_FILE_DATA, TSDB_FILE_LAST, TSDB_FILE_MAX, TSDB_FILE_META } TSDB_FILE_T;
```

Comment:
The file TSDB_FILE_META_TMP is not neccessary. You can use a new versioned meta file, For example: meta-ver1. This trick is used in TSDB file sync now.

Reply:
Great. Then there will be less one conflict for me!

Reply:
thanks, fixed it